File: web3rpc/rpc-specs/code-samples/python/debug/vMStandardTracing/standardTraceBlockToFile.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

blockHash = "0x1d5ba00e313a81ae6d409d459c153327072665d9ea2f47608369722baf0cfbb6"

w3 = Web3(Web3.HTTPProvider(host))
debug_response = w3.debug.standard_trace_block_to_file(blockHash)

print(debug_response)

File: web3rpc/rpc-specs/code-samples/javascript/debug/vMStandardTracing/standardTraceBlockToFile.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
let sdk = new JsonRpcProvider("https://public-en-baobab.klaytn.net");
    const blockHash = "0xdabbc6a0d2619c56db4645c4e85799af9927bdf3bd13d0c77e49db413e3db9f3"
    
    sdk.debug.standardTraceBlockToFile(blockHash, {}, (err, data, response) => {
        console.log(data);
    });
}
)()